packages/ckeditor5-font/theme/fontcolor.css

```diff
@@ -3,19 +3,43 @@
  * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
  */
 
-.ck .ck-button.ck-color-table__remove-color {
+.ck .ck-button.ck-color-table__remove-color,
+.ck .ck-button.ck-color-table__color-picker {
 	display: flex;
 	align-items: center;
 	width: 100%;
```

Comment:
Please add here also a mixin `ck-dir rtl` and set `justify-content: flex-start;` to make buttons looks ok in RTL mode :)

<img width="200" alt="Screenshot 2023-04-17 at 15 11 16" src="https://user-images.githubusercontent.com/7074833/232494187-8fb4607f-c12f-425e-a03f-13471e06ee30.png">